kornia/filters/sobel.py

```diff
@@ -28,16 +37,29 @@ class SpatialGradient(nn.Module):
         >>> output = kornia.filters.SpatialGradient()(input)  # 1x3x2x4x4
     """
 
-    def __init__(self) -> None:
+    def __init__(self, mode: str = 'Sobel') -> None:
```

Comment:
I would use lower case -> `sobel` in order to be more consistent with pytorch standards